content/events/hacktoberfest.adoc

```diff
@@ -14,56 +14,52 @@ opengraph:
+Generally, any four non-spammy pull requests in GitHub or merge requests in GitLab would qualify.
+Please see the link:/participate/[Contribute and Participate] page for more information about how to contribute.
 
 == Quick start
 
 1. Sign-up to Hacktoberfest on link:https://hacktoberfest.com[the event website].
-2. Join link:https://app.gitter.im/#/room/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
-3. Everything is set, just start creating pull-requests!
-** If a repository has no `hacktoberfest` topic set, contact the maintainers to see if they are willing to accept a contribution.
-
+2. Join link:https://matrix.to/#/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
+3. Everything is set, just start creating pull or merge requests!
+** If a repository has no `hacktoberfest` topic is set, contact the maintainers to see if they are willing to accept a contribution.
+** Alternatively, if a pull or merge request has the label `hacktoberfest-accepted`, it will count towards your total.
+** Your pull or merge request must not be labeled as invalid to be counted.
+** For your pull or merge requests to be accepted, your pull or merge requests must be merged, or have the `hacktoberfest-accepted` label, or have an overall approving review on GitHub.
 
 == Where to contribute?
 
-The Jenkins project is spread across several organizations on GitHub (`link:https://github.com/jenkinsci/[jenkinsci]`, `link:https://github.com/jenkins-infra/[jenkins-infra]`, `link:https://github.com/jenkins-zh/[jenkins-zh]`).
+The Jenkins project is spread across several organizations on GitHub (`link:https://github.com/jenkinsci/[jenkinsci]` and `link:https://github.com/jenkins-infra/[jenkins-infra]`.).
 You are welcome to contribute to **any** repository in **any** of those organizations, or any other Jenkins-related repository on GitHub.
 Repositories may have different contribution guidelines, review and merge policies.
-If you adopt Jenkins in your open-source projects (e.g. Jenkins Pipeline or Configuration as Code),
-it counts as well!
+If you adopt Jenkins in your open-source projects (e.g. Jenkins Pipeline or Configuration as Code), it counts as well!
 Note that not all pull requests will automatically count towards Hacktoberfest.
 
-* link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+org%3Ajenkins-zh+topic%3Ahacktoberfest[List of repositories marked for Hacktoberfest] -
-  you can just submit pull requests, no extra steps needed.
-* for other repositories, we will need to get the repo marked for Hacktoberfest so that your pull request counts towards the goal.
+* link:https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+topic%3Ahacktoberfest&type=repositories[List of repositories marked for Hacktoberfest] - you can just submit pull requests, no extra steps needed.
+* For other repositories, we will need to get the repo marked with the topic "Hacktoberfest" for your pull request to count towards the goal.
```

Comment:
```suggestion
* For other repositories, we will need to get the repo marked with the "Hacktoberfest" label for your pull request to count towards the goal.
```
I _think_ it's considered a label, but I could be wrong.

Reply:
For repositories it is a "topic" while PRs and issues get a "label" in Github terminology AFAIK.

Reply:
Ahh, that makes more sense, that's where my confusion was. thanks very much @jmMeessen !